modules/distr-tracing-tempo-config-storage.adoc

```diff
@@ -0,0 +1,728 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: REFERENCE
+[id="distr-tracing-tempo-config-storage_{context}"]
+= Distributed tracing storage configuration options
+
+You configure storage for the Collector, Ingester, and Query services under `spec.storage`. Multiple instances of each of these components can be provisioned as required for performance and resilience purposes.
+
+.General storage parameters used by the {TempoName} Operator to define distributed tracing storage
+
+[options="header"]
+[cols="l, a, a, a"]
```

Comment:
This config is for Jaeger. Need revision

Reply:
@rubenvp8510, could you please review this and add a suggested change here?

` ```suggestion`

` ``` `